SnapXEats/SnapXEats/Assets.xcassets/Directions/marker_icon.imageset/Contents.json

```diff
@@ -2,17 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "Mark.png",
+      "filename" : "ico-mark copy 2.png",
```

Comment:
remove 'copy 2' from file name

Reply:
These are the names of actual Image files used. They won't be visible anywhere. In our code, we will access them by using "marker_icon" only.